gems/pending/Scvmm/miq_hyperv_disk.rb

```diff
@@ -150,7 +150,7 @@ def bread(start_sector, number_sectors)
       encoded_data = @parser.output_to_attribute(run_correct_powershell(read_script))
       t2           = Time.now.getlocal
       buffer       = ""
-      Base64.decode64(encoded_data).split(' ').each { |c| buffer += c.to_i.chr }
+      Base64.decode64(encoded_data).split(' ').each { |c| buffer << c.to_i.chr }
```

Comment:
Maybe you should use the `ToBase64Transform` class:
https://msdn.microsoft.com/en-us/library/system.security.cryptography.tobase64transform%28v=vs.110%29.aspx